mavros/src/lib/uas_data.cpp

```diff
@@ -53,6 +53,7 @@ UAS::UAS() :
 
 	// send static transform from "local_origin" (ENU) to "local_origin_ned" (NED)
 	publish_static_transform("local_origin", "local_origin_ned", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, M_PI_2)));
+	publish_static_transform("fcu", "fcu_frd", Eigen::Affine3d(ftf::quaternion_from_rpy(M_PI, 0, 0)));
```

Comment:
we actually considered this on https://github.com/mavlink/mavros/blob/master/mavros/src/lib/ftf_frame_conversions.cpp#L32-L37. So I was feeling strange that you were not considering this here also, since for vision plugin, that is also considered: https://github.com/mavlink/mavros/blob/master/mavros_extras/src/plugins/vision_pose_estimate.cpp#L127-L129

Reply:
can we actually call them `body_nwu` and `body_ned` instead?

Reply:
Yeah, that's fine with me.

Reply:
@jgoppert please update the PR with the naming convention above

Reply:
We will keep the convention as is.